about.html

```diff
@@ -0,0 +1,154 @@
+  </head>
+  <body>
+    <div>
+      <div id="container">
+        <div class="nav-bar">
+          <nav class="primary-nav">
+            <ul class="primary-nav">
+              <li>
+                <a href="#"><i class="fab fa-facebook"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fab fa-twitter"></i></a>
+              </li>
+              <li>
+                <a href="#" class="my-page"><p>MyPage</p></a>
+              </li>
+              <li>
+                <a href="#" class="logout log-pad"><p>Logout</p></a>
+              </li>
+            </ul>
+          </nav>
+          <nav class="secondary-nav">
+            <div class="banner secondary-nav">
+              <a href="index.html"
+                ><img
+                  class="logo-img"
+                  src="images/eagle-logo.jpeg"
+                  alt="logo"
+                />
+                <p class="comp-name">Haddis Tecnology</p></a
+              >
+            </div>
+            <ul class="secondary-nav nav-menu">
+              <li class="nav-item">
+                <a href="index.html" class="link-nav home-link">Home</a>
+              </li>
+              <li class="nav-item">
+                <a href="about.html" class="link-nav">About</a>
+              </li>
+              <li class="nav-item"><a href="" class="link-nav">Program</a></li>
+              <li class="nav-item"><a href="" class="link-nav">Join</a></li>
+              <li class="nav-item"><a href="" class="link-nav">Sponsor</a></li>
+              <li class="nav-item"><a href="" class="link-nav">News</a></li>
+              <li class="nav-item">
+                <a href="" class="confference">Confference</a>
+              </li>
+            </ul>
+          </nav>
+
+          <div class="hamburger" id="hamburger">
+            <span class="l-1 bar"></span>
+            <span class="l-2 bar"></span>
+            <span class="l-3 bar"></span>
+          </div>
+        </div>
+      </div>
+      <section class="headlines">
+        <div class="advert-txt">
+          <h1>Creative Idea Discustions 2022</h1>
+        </div>
+        <div class="box-txt">
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, ad
+            necessitatibus nesciunt voluptate numquam odit veniam reprehenderit
+            expedita doloremque laboriosam natus eligendi itaque, maxime
+            corporis dolorum ipsa neque deserunt. Voluptatem. Lorem ipsum dolor
+            sit amet consectetur adipisicing elit. Iure, ad necessitatibus
+            nesciunt voluptate numquam odit veniams
+          </p>
+        </div>
+        <div class="address-txt">
+          <p>
+            Please contact us per email for further quetsions about our campany!
+          </p>
+          <br />
+          <a href="#">kumilachew1212@gmail.com</a>
+        </div>
+        <div class="logo-container">
+          <h3 class="logo-desk">Our History</h3>
```

Comment:
- [ ] Please add the horizontal orange lines under all of your headings in the About page like in the Behance design.
> Your Design

![image](https://user-images.githubusercontent.com/90556221/181484072-2415768c-544c-4d58-bc93-d8f165c67290.png)

> Behance
![image](https://user-images.githubusercontent.com/90556221/181484184-ff2b31f8-b254-4e94-8869-17989533f7af.png)